src/applications/disability-benefits/781/pages/informationInterviewCombat.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { IncidentIntroduction781 } from '../helpers';
+
+export const uiSchema = {
+  'ui:description': ({ formData }) => (
+    <IncidentIntroduction781 formData={formData} formType="781"/>
```

Comment:
I think this is a good solution. Was wondering what you though about my idea though @imeecuison: https://github.com/department-of-veterans-affairs/vets-website/pull/8603/files#r222069185